src/app/pages/payment/payment.routing.ts

```diff
@@ -0,0 +1,10 @@
+import { TokenizedCardComponent } from './tokenized-card.component';
+import { PinValidateComponent } from './pin-validate.component';
+import {Routes} from '@angular/router';
+import { PinPaymentComponent } from './pin-payment.component';
+
+export const paymentRoutes: Routes = [
+	{path: 'pin', component: PinPaymentComponent},
+	{path: 'validate-pin/:flwRef/:purpose', component: PinValidateComponent},
+	{path: 'tokenized-card', component: TokenizedCardComponent},
```

Comment:
Hey @Victoradukwu, our current URLs are not constructed properly, we could group all payment URLs  in the payments namespace such that it's like `/payments/pay/tokenized-card`

Reply:
Thanks, @CryceTruly . The `URL`s have been renamed.